content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -45,8 +45,13 @@ kubectl create namespace qos-example
 
 For a Pod to be given a QoS class of Guaranteed:
 
-* Every Container, including init containers, in the Pod must have a memory limit and a memory request, and they must be the same.
-* Every Container, including init containers, in the Pod must have a CPU limit and a CPU request, and they must be the same.
+* Every Container in the Pod must have a memory limit and a memory request.
+* For every Container in the Pod, the memory limit must equal the memory request.
+* Every Container in the Pod must have a CPU limit and a CPU request.
+* For every Container in the Pod, the CPU limit must equal the CPU request.
+
+These restrictions apply to init containers and app containers equally.
+* Every Container, including init containers, in the Pod must have a CPU limit and a CPU request, and the two values must be the same.
```

Comment:
Oops - is this line still meant to be here?

Reply:
```suggestion
```

Ah, it is not